File: PYTHON/Program/Human.py
#HUMAN
class Human:
    __nama = ""
    __nik = ""
    __jenis_kelamin = ""

    def __init__(self, nik = "", nama = "", jenis_kelamin =""):
        # Set attributes pada class mahasiswa
        self.__nama = nama
        self.__nik = nik
        self.__jenis_kelamin = jenis_kelamin

    # Getter dan Setter

    # Getter dan setter untuk nama mahasiswa
    def get_nama(self):
        return self.__nama
    def set_nama(self, nama):
        self.__nama = nama

    # Getter dan setter untuk nik mahasiswa
    def get_nik(self):
        return self.__nik
    def set_nik(self, nik):
        self.__nik = nik
        
    # Getter dan setter untuk nama mahasiswa
    def get_jenis(self):
        return self.__jenis_kelamin
    def set_jenis(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

File: CPP/Program/Human.cpp
#include "Header.hh"

//HUMAN
Human :: Human()
{
    nik = "";
    name = "";
    gender = '-';
}

Human :: Human(string nik, string name, char gender)
{
    this->nik = nik;
    this->name = name;
    this->gender = gender;
}

string Human::getNik()
{
    return nik;
}

void Human ::setNik(string nik)
{
    this->nik = nik;
}

string Human::getName()
{
    return name;
}

void Human ::setName(string name)
{
    this->name = name;
}

char Human::getGender()
{
    return gender;
}

void Human ::setGender(char gender)
{
    this->gender = gender;
}

Human::~Human()
{

}